lib/suspenders/generators/app_generator.rb

```diff
@@ -114,6 +114,7 @@ def setup_production_environment
       build :enable_rack_canonical_host
       build :enable_rack_deflater
       build :setup_asset_host
+      build :create_heroku_pipelines_config_file
```

Comment:
The location where this is called seems a bit weird to me. Is it only needed in `production` or do we need it in the base app? It might make more sense to place it under `suspenders_customization` rather than `setup_production_environment` section.